templates/login.html

```diff
@@ -1,19 +1,20 @@
 {% extends 'auth-base.html' %}
-
 {% block content %}
-<form class="form-signin" method="POST" action="/meeting/login">
+    <canvas id="canvas"></canvas>
+<div class = "fixed-top">
+<form class="form-signin" method="POST" action="/meeting/login" >
   {% csrf_token %}
-  <h1 class="h3 mb-3 font-weight-normal">Sign In</h1>
-  
+  <h4 class="h3 mb-3 font-weight-normal" style= "font-family: 'Rancho', cursive;" >Sign In</h4>
+
   <input type="text" name="inputUsername" class="form-control" placeholder="User Name" required autofocus>
   <input type="password" name="inputPassword" class="form-control" placeholder="Password" required>
   
   {% if errors %}
     <p style="color: red">Invalid Username and password combination.</p>
   {% endif %}
-  
   <button class="btn btn-lg btn-primary btn-block" type="submit">Sign in</button>
-  <a class="btn btn-lg btn-primary btn-block" href="/meeting/register">Register</button>
+    <a class="btn btn-lg btn-primary btn-block" href="/meeting/register">Register</button></a>
```

Comment:
Maybe he only wanted the anchor on the second half of the button.

Reply:
Haha! But in all seriousness, just for the context: https://www.w3.org/TR/html4/intro/sgmltut.html (section 3.2.1), and we'll end up running into inconsistent issues with improperly-nested code, especially when CSS selectors are used.

Reply:
Actually if you look at the code the tags are originally it was <a class> </button> 
there was no beginning tag of <button> so pycharm was saying it was missing a closing tag so I added the tag and it worked. of course if you  actually read it i just needed to change the a to button. HAHA

Reply:
Ah yes, my mistake. I guess it was just missing the starting tag entirely then. Same difference though since it still isn't nested properly--when that fix is incorporated I'll get it merged in.